core/os/device/abi.go

```diff
@@ -25,9 +25,10 @@ var (
 	AndroidMIPS     = abi("mips", Android, MIPS, Little32)
 	AndroidMIPS64   = abi("mips64", Android, MIPS64, Little64)
 
-	LinuxX86_64   = abi("linux_x64", Linux, X86_64, Little64)
+	LinuxX86_64   = abi("linux_x64", OSKind_Linux, X86_64, Little64)
```

Comment:
Nit: Slightly odd change here?

Reply:
Thanks for catching it, I had removed "Linux" so I could catch all of the places we were using it, forgot to put this one back.